### PR TITLE
LPD-38858 Refator poshi test in playwright

### PR DESCRIPTION
**Pull request by michelevigilante**
https://liferay.atlassian.net/browse/LPD-38858

### CODE REVIEW

**Comment by michelevigilante** [comment]
ci:test:sf

File: modules/test/playwright/tests/commerce/commerce-product-content-web/productCard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+	const product = (
+		await apiHelpers.headlessCommerceAdminCatalog.getProducts(
+			new URLSearchParams({
+				filter: `name eq 'U-Joint'`,
+			})
+		)
+	).items[0];
+
+	const productSkus = await apiHelpers.headlessCommerceAdminCatalog
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/tests/commerce/commerce-product-content-web/productCard.spec.ts:76]: I would check the UI not the API to make sure the buyer can view the price, this doesn't certify the UI is showing the right price.

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by michelevigilante** [comment]
ci:test:sf

**Comment by michelevigilante** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:playwright:no-compile

**Comment by andrea-ale-sbarra** [comment]
ci:test:playwright:nocompile

**Review by andrea-ale-sbarra — COMMENTED** [review]
@michelevigilante test is failing on ci

**Comment by michelevigilante** [comment]
ci:test:playwright:nocompile

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**